.github/workflows/build.yml

```diff
@@ -1066,3 +1066,51 @@ jobs:
             # For Pico, Beta channels can only get one build, not a separate China / non-China build
             ./pico-cli upload-build --app-id $PICO_APP_ID --app-secret $PICO_APP_SECRET --region noncn --apk OpenBrush_Pico_$VERSION.apk --channel 3 --notes-en "Version $VERSION" --device 'PICO Neo3,PICO Neo3 Pro,PICO Neo3 Eye,PICO 4'
           fi
+
+  publish_appstore:
+    name: Release to the App Store
```

Comment:
Please change this to "iOS App Store", so we don't confuse it with the Mac app store.